app/jobs/course/assessment/answer/auto_grading_job.rb

```diff
@@ -2,6 +2,8 @@
 class Course::Assessment::Answer::AutoGradingJob < ApplicationJob
   include TrackableJob
 
+  retry_on PriorityShouldBeLoweredError, wait: 2.seconds, queue: :delayed_highest
```

Comment:
any reason why you add `wait: 2.seconds`?

Reply:
this can be any number, but if we don't set it up, the default waiting time is 3 seconds, which is quite long in my opinion.

Reply:
2.seconds and 3.seconds are immaterial, I rather either `wait: 0`, or not put anything at all.

Reply:
done. I decided to not put anything over there